src/main/java/amara/command/AddToDoCommand.java

```diff
@@ -0,0 +1,39 @@
+package amara.command;
+
+import java.util.ArrayList;
+
+import amara.storage.Storage;
+import amara.task.Task;
+import amara.task.ToDo;
+import amara.ui.Ui;
+
+/**
+ * A {@link Command} implementation that adds a {@link ToDo} task
+ * to a given {@code ArrayList<Task>}.
+ * <p>
+ * The given {@link ToDo} task is  added to the provided {@code ArrayList<Task>}.
+ * </p>
+ */
+public class AddToDoCommand extends Command {
+    private static final String MESSAGE = "Got it. I've added this task:\n"
+            + "  %s\nNow you have %d tasks in the list.";
+    private ToDo todo;
```

Comment:
```suggestion
    private ToDo toDo;
```
Coding standard stuff: variable name should be camelCase